src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -312,8 +326,8 @@ public static void checkApiUrlValidity(@Nonnull GitHub gitHub, @CheckForNull Sta
             StandardUsernamePasswordCredentials c = (StandardUsernamePasswordCredentials) credentials;
             hash = Util.getDigestOf(c.getPassword().getPlainText() + SALT);
         } else {
-            // TODO OAuth support
```

Comment:
Still need the `else` (same as https://github.com/jenkinsci/github-branch-source-plugin/pull/269/files#diff-61948c4cd4e480f18c6e572cf88a055bR372)